cmd/earthly/main.go

```diff
@@ -1219,6 +1219,100 @@ Set up a whole custom git repository for a server called example.com, using a si
 						},
 					},
 				},
+				{
+					Name:        "secret",
+					Aliases:     []string{"secrets"},
+					Description: "Manage cloud secrets *experimental*",
+					Usage:       "Manage cloud secrets *experimental*",
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "org",
+							EnvVars:  []string{"EARTHLY_ORG"},
+							Usage:    "The organization to which the project belongs.",
+							Required: true,
+						},
+						&cli.StringFlag{
+							Name:     "project",
+							EnvVars:  []string{"EARTHLY_PROJECT"},
+							Usage:    "The organization project in which to store secrets.",
+							Required: true,
+						},
+					},
+					Subcommands: []*cli.Command{
+						{
+							Name:  "set",
+							Usage: "Stores a secret in the secrets store",
+							UsageText: "earthly [options] secret set <path> <value>\n" +
+								"   earthly [options] secret set --file <local-path> <path>\n" +
+								"   earthly [options] secret set --file <local-path> <path>",
```

Comment:
should one of these be an example of `--stdin` instead? (currently there's two examples for `--file` which appear to be the same).

Reply:
Oh, good point. I actually copied this from the existing commands, so I'll fix it in both places.